.github/workflows/greetings.yml

```diff
@@ -12,5 +12,5 @@ jobs:
     - uses: actions/first-interaction@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        issue-message: 'Thank you for submitting an issue to the Mirantis Terraform Provider'
-        pr-message: 'Thank you for contributing to the Mirantis Terraform Provider'
+        issue-message: 'Thank you for submitting an issue to the Mirantis Launchpad Provider'
+        pr-message: 'Thank you for contributing to the Mirantis Launchpad Provider'
```

Comment:
Interesting why it says: Mirantis Launchpad Provider, instead of Mirantis MKE Provider? o.O

Reply:
probably let over from the rewrite. Let me look

Reply:
fixed